src/server/system_services/schemas/nsfs_bucket_schema.js

```diff
@@ -29,10 +28,6 @@ module.exports = {
         system_owner: {
             type: 'string',
         },
-        // bucket_owner is the account name
-        bucket_owner: {
```

Comment:
for backward compatibility we will need to keep that

Reply:
This would leave us with issues 8080 and 7793.